### DISCUSSION TITLE
Alternative to Elastic Search (with MySQL protocol)

### DISCUSSION THREAD

**Discussion by nosovk** [pimcore/pimcore#13429] [Ideas]
We often have to build some full text search over data in PIMCore. The standard way proposed in the documentation is to install Elasticsearch and feed it with our Data. It's a common way, but it's a bit complicated from a developer\support perspective.
I suggest taking a look at https://github.com/manticoresoftware/manticoresearch/
It's a fork of Sphinx search engine, which was popular before Elasticsearch surpassed all other solutions.
It can use MySQL protocol for requests, it means that there is no need for additional dependencies, it could be easily adopted and used with existing codebase. Also, it's easy to index big mysql\mariadb with Manticore. 
In common, it could provide much better user experience due to solutions similar to base tech stack of PIMCore.

**Comment by BlackbitDevs** [comment]
Will take a look on this in context of https://github.com/pimcore/pimcore/issues/13689 - maybe this is an even better approach than MySQL ngram fulltext search.
